src/sections/SMP/index.js

```diff
@@ -117,16 +121,16 @@ const SMPPage = () => {
                         </iframe>
                         <iframe	className="iframe" src="https://docs.google.com/presentation/d/e/2PACX-1vSWhREOZIfJqTG8jHtitgb5e-OAHC7E8EMmoVCaTRmC8b_itHkeFUNKcBVn9oaTpX5ozYvlhscvBXiK/embed?start=false&loop=false&delayms=3000"	
                             frameBorder="0" allowFullScreen>	
-                        </iframe>	
-                        {/* </Row> */}
+                        </iframe>
                     </div>
                 </div>
                 <div className="cncf-callout">
                     <h1> Join the CNCF Service Mesh Working Group </h1>
                     <h4> Learn more about these initiatives</h4>
-                    <Button primary title="Learn More"></Button>
+                    <Button primary title="Learn More" url="https://smp-spec.io"></Button>
                 </div>
                 <div className="layer5-callout">
```

Comment:
The `Join-Community` code in `src/sections/Community/Join-community` can be used here instead of re-writing the code. You can adjust the width for this page by redefining the changes using the className of that component for this page.